### ISSUE TITLE
Sample data to perform few-shot training

### ISSUE THREAD

**Issue by darsh10** [erikalu/class-agnostic-counting#5]
Hey Authors,

I very recently learned about this implementation and I find it very exciting for medical research applications. Would it be possible for the authors or someone to share what sample (fine-tuning) training data looks like?

Thanks,
Darsh.

**Comment by erikalu** [comment]
Hi Darsh, if I'm understanding you correctly, and you want an example of the labeled data we used for fine-tuning the model, take a look at the cell dataset here: http://www.robots.ox.ac.uk/~vgg/software/cell_detection/. We used it for one of our fine-tuning experiments.

**Comment by darsh10** [comment]
Hi Erika,

I am actually wondering how I can use my own (custom) labels. Is that
possible? I am fine if I need to train a model from scratch.

Thank you,
Darsh.

**Comment by erikalu** [comment]
Yes, you can use your own custom data to finetune/adapt the pretrained model. Just download the pretrained model weights and set up your custom dataset in the same way as the example datasets. For minimal changes to the code, you'll want to modify src/data_loader.py and also create a new .npz file with the filenames. Please take a look at that data_loader.py file to see what is expected of the dataset format.
